cvat-ui/src/components/annotation-page/standard3D-workspace/styles.scss

```diff
@@ -171,3 +171,96 @@
 .cvat-control-side-bar-icon-size {
     font-size: $grid-unit-size * 5;
 }
+
+.cvat-canvas3d-perspective {
+    height: 50%;
+    width: 100%;
+    position: relative;
+    padding: $grid-unit-size/2;
+}
+
+.cvat-canvas3d-orthographic-views {
+    height: 50%;
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+}
+
+.cvat-canvas3d-perspective-arrow-directions {
+    position: absolute;
+    padding: $grid-unit-size;
+    bottom: 0;
+    right: 0;
+}
+
+.cvat-canvas3d-perspective-arrow-directions-icons-up {
+    margin-left: $grid-unit-size * 5.25;
+}
+
+.cvat-canvas3d-perspective-arrow-directions-icons-bottom {
+    margin: $grid-unit-size/2;
+}
+
+.cvat-canvas3d-perspective-arrow-directions-icons-color {
+    color: black;
+}
+
+.cvat-canvas3d-perspective-directions {
+    padding: $grid-unit-size;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+}
+
+.cvat-canvas3d-perspective-directions-icon {
+    margin: $grid-unit-size/2 $grid-unit-size/4;
+    width: $grid-unit-size * 4;
+}
+
+.cvat-canvas3d-topview {
+    width: 33.333%;
+    height: 100%;
```

Comment:
width and height is a common for top side and front move it with single class, or make a mixin for this stuff

Reply:
Resolved